scripts/tests.sh

```diff
@@ -30,6 +30,17 @@ set -e
 
 REPO_ROOT="$(dirname "$0")"/..
 
+smt=""
+ipc=""
```

Comment:
I think it makes more sense for these two to be named `ENABLE_IPC` and `ENABLE_SMT` and checking for `true / false`.

Reply:
I see its not consistent, mostly the for loop uses lowercase ones. Please just check other scripts what is the preferred way. I am not sure which one is, I remember seeing uppercase more frequently.

Reply:
Looking at the other scripts, ``UPPER_CASE`` seems to be the default.

Introducing ``ENABLE_IPC`` and ``ENABLE_SMT`` again, instead of working on the ``soltest`` flags directly, would need more code to set these flags based on the above. 
On the other hand it makes it more readable. @chriseth proposed to use ``smt`` and ``ipc`` directly, which I do like as well. It would also be consistent with how other flags are being handled.

Reply:
Here is an example for this:

```
progress="--show-progress"
if [ "$CIRCLECI" ]
then
    progress=""
fi
```

Reply:
Yep, but IPC is special because its on by default and the flag is a negation. In the end having `-z $ipc` (means: empty ipc) in the checks for running/killing `testeth` is very misleading.

Reply:
Well, this is a very good point!